src/pages/index.js

```diff
@@ -165,42 +239,117 @@ const cardImageEl = document.querySelector(".card__image");
 const cardCaptionEl = document.querySelector(".card__title");
 
 newPostBtn.addEventListener("click", function () {
+  const inputList = Array.from(
+    newPostForm.querySelectorAll(validationConfig.inputSelector)
+  );
+
+  toggleButtonState(
+    inputList,
+    newPostForm.querySelector(validationConfig.submitButtonSelector),
+    validationConfig
+  );
+
   openModal(newPostModal);
 });
 
 newPostCloseButton.addEventListener("click", function () {
   closeModal(newPostModal);
 });
 
-function handleEditProfileSubmit(evt) {
-  evt.preventDefault();
-  profileNameEl.textContent = editProfileNameInput.value;
-  profileDescriptionEl.textContent = editProfileDescriptionInput.value;
-  closeModal(editProfileModal);
-}
+avatarModalBtn.addEventListener("click", function () {
+  const inputList = Array.from(
+    avatarForm.querySelectorAll(validationConfig.inputSelector)
+  );
 
-editProfileForm.addEventListener("submit", handleEditProfileSubmit);
+  toggleButtonState(
+    inputList,
+    avatarForm.querySelector(validationConfig.submitButtonSelector),
+    validationConfig
+  );
 
-function handleNewPostSubmit(evt) {
-  evt.preventDefault();
+  openModal(avatarModal);
+});
 
-  const inputValues = {
-    name: newPostCaptionInput.value,
-    link: newPostImageInput.value,
-  };
+avatarCloseButton.addEventListener("click", function () {
+  closeModal(avatarModal);
+});
 
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  closeModal(newPostModal);
-  newPostForm.reset();
-  disableButton(cardSubmitButton, validationConfig);
-}
+avatarForm.addEventListener("submit", (evt) => {
+  evt.preventDefault();
+  const originalText = avatarSubmitButton.textContent;
+  avatarSubmitButton.textContent = "Saving...";
+
```

Comment:
* Please, make the `inputs` match the `mobile` design on 320px https://snipboard.io/NCLPpR.jpg